e2e/scenarios/kops-simple

```diff
@@ -86,7 +86,9 @@ if [[ -z "${ADMIN_ACCESS}" ]]; then
 fi
 echo "ADMIN_ACCESS=${ADMIN_ACCESS}"
 
-create_args="--networking gce"
+# cilium does not yet pass conformance tests (shared hostport test)
```

Comment:
Not sure how we're getting around this elsewhere... this is the seemingly well-known https://github.com/cilium/cilium/issues/14287